src/arguments/utility.py

```diff
@@ -58,10 +60,20 @@ def evoke_search_error(self, error_statement):
 
 
 class Utility():
+
     def __init__(self):
         # the parent url
         self.search_content_url = "https://api.stackexchange.com/"
 
+    def clear(self):    # function to clear Screen
+        # for windows 
+        if name == 'nt': 
+            _ = system('cls') 
+      
+        # for mac and linux(here, os.name is 'posix') 
+        else: 
+            _ = system('clear') 
+
```

Comment:
You have added a lot of spaces in between. You can replace it with:

```python
def clear(self):
    """ Clear the screen based on the operating system """
    _ = system("cls") if os.name == "nt" else system("clear")

```

Reply:
I tried this but it wasn't working. I removed the spaces and made it as as compact as possible